src/userApp/verifyToken.js

```diff
@@ -0,0 +1,8 @@
+import jwt from 'jsonwebtoken';
```

Comment:
What does this file do?
If it is related to the token generation it can be safely removed since we are using `uuid` to generate the token now.